Marlin/Configuration.h

```diff
@@ -1564,4 +1564,59 @@
   //#define FILAMENT_LCD_DISPLAY
 #endif
 
+//===========================================================================
+//============================ I2C Encoder Settings =========================
+//===========================================================================
+
```

Comment:
Maybe more commentary about what this is, how it works both in theory and practically, that it's meant to be used with encoder boards using the '5311, details on their resolution, accuracy, etc.  Technical details would help, generally, I think.